### ISSUE TITLE
Enable/Disable Package Startup messages

### ISSUE THREAD

**Issue by pgensler** [trinker/pacman#99]
Would it be possible to have a flag in `p_load` to enable/ disable package startup messages? I noticed when using dplyr with p_load, no messages appear. Is this a bug, or is that behavior intentional?

**Comment by Dasonk** [comment]
I believe it was our intention as we find them noisy. At one point I thought we were going to add in a parameter to give the user more control but apparently never got around to it.

**Comment by trinker** [comment]
@Dasonk yes I thought there was an arg that made the install more or less verbose.  If not we indeed did discuss that this would be the case and should be implemented.

**Comment by Dasonk** [comment]
I think we talked about a `silent` parameter at some point. But thinking back I'm not sure if we even included package startup messages in our consideration and that was more or less to control the messages we were going to print about which packages loaded successfully or unsuccessfully.  Either way i think we agree that having better control over the information printed would be a good thing and I'll attach a feature request tag to this.

**Comment by pgensler** [comment]
Hi Dasonk, I agree that it might be helpful to have a ```silent``` parameter default as true, and let the user change if necessary. Thanks for the consideration.
